android-wsbridge/app/build.gradle

```diff
@@ -40,28 +41,32 @@ android {
             buildConfigField "String", "REMOTE_LOGGER_URL", "\"your_url_goes_here.com\""
         }
     }
+
 }
 
 dependencies {
+
     implementation 'com.amitshekhar.android:android-networking:0.3.0'
     implementation 'org.java-websocket:java-websocket:1.3.3'
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'com.squareup:otto:1.3.8'
-    implementation('com.crashlytics.sdk.android:crashlytics:2.6.5@aar') {
-        transitive = true;
-    }
     implementation 'io.reactivex.rxjava2:rxandroid:2.0.2'
     // Because RxAndroid releases are few and far between, it is recommended you also
     // explicitly depend on RxJava's latest version for bug fixes and new features.
     implementation 'io.reactivex.rxjava2:rxjava:2.1.12'
-    implementation 'com.android.support:support-annotations:25.0.1'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.5.1'
-    internalImplementation 'com.squareup.leakcanary:leakcanary-android:1.5.1'
-    releaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
-    debugImplementation 'com.github.markzhai:blockcanary-android:1.5.0'
-    internalImplementation 'com.github.markzhai:blockcanary-android:1.5.0'
-    releaseImplementation 'com.github.markzhai:blockcanary-no-op:1.5.0'
+    implementation 'androidx.annotation:annotation:1.1.0'
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'androidx.core:core:1.3.2'
+
+    implementation 'com.github.markzhai:blockcanary-android:1.5.0'
```

Comment:
Do we still need this? I see some blockcanary code was deleted in the BridgeApplication

Reply:
we still have a class using it.